procfile/README.md

```diff
@@ -78,3 +78,13 @@ This allows you to set process specific environment variables. If these are set
 environment:
   EMPIRE_X_LOAD_BALANCER_TYPE: "alb"
 ```
+
+Supported environment variables that can either be set via `emp set` for the whole application or
+inside the `Procfile` for a specific process.
+
+Name | Default value | Available options | Description
+-----|---------------|-------------------|------------
+`EMPIRE_X_LOAD_BALANCER_TYPE` | `elb` | `alb`, `elb`| Determines whether you will use an ALB or ELB
+`EMPIRE_X_EXPOSURE` | `private` | `private`, `public` | Sets whether your ALB or ELB will be public (internet-facing) or private (internal), the default is private, however if you have used the deprecated `domain-add` command then the load balancer will become public. **If you change this setting, the load balancer will be recreated as soon as you deploy**
```

Comment:
One thing that we've tried to do with the extended Procfile is maintain some level of "parity" with `docker-compose.yml`, so that concepts just kinda translate over (e.g. ports).

In docker-compose.yaml, [you can attach a container to a "network"](https://docs.docker.com/compose/compose-file/#networks) (like `external` or `internal`), which I think maps pretty nicely to how we use internal/external ELB's. I wonder if we should just use a `networks` key to determine whether the process should be exposed external/internal.

For example:

```yaml
web:
  command: ./bin/web
  networks:
    - external # Expose this process with an "external" ELB.
admin:
  command: ./bin/web
  networks:
    - internal # Expose this process with an "internal" ELB
```

Just a thought, totally fine with starting with some `EMPIRE_X` env vars.